### ISSUE TITLE
Error with SASS build

### ISSUE THREAD

**Issue by katyaugust28** [jaumefontal/SASS-Build-SublimeText2#39]
The following was printed at the bottom of the screen:

[Errno 2] No such file or directory: 'sass'
[cmd: ['sass', '--update', '/Users/katyaugust/Desktop/Portfolio/scss/styling.scss:/Users/katyaugust/Desktop/Portfolio/scss/styling.css', '--stop-on-error', '--no-cache']]
[dir: /Users/katyaugust/Desktop/Portfolio/scss]
[path: /usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin]
[Finished]

**Comment by Meloninga54** [comment]
**TLDR: This package was made for Ruby Sass which not supported anymore, so you'll have to copy the source into your own build-system and remove the --no-cache option from the [sublime-build file](https://github.com/jaumefontal/SASS-Build-SublimeText2/blob/master/SASS.sublime-build)**.

## Step-By-Step Solution
I know this is a bit late, but I've just spent four soul-crushing hours figuring out the solution to this really easy problem, and I don't want anyone else to go through the same thing.

### The problem

If you're facing this problem, you've installed Dart Sass, which is a **good thing**. Ruby Sass isn't supported anymore. 

Basically, this package contains [sublime-build](https://www.sublimetext.com/docs/build_systems.html) files, which contain executable commands to convert sass to css. Ruby Sass had a command line option called [--no-cache](https://sass-lang.com/documentation/cli/ruby-sass#no-cache) which is hard coded into this package's compilation commands. Long story short, Dash Sass doesn't support this flag, so we have to remove it.

### How to solve it

Basically, you'll have to make your own build file without the  --nocache flag. **This is easy.**

1. **In Sublime, go to Tools -> Build System -> New Build System.** This should create a new file calle

**Comment by wi0lono** [comment]
Edit: Before trying this, make sure you have installed Sass, and that it's accessible from the command line. If not, ignore everything below and make sure it's visible from there.

**TLDR: This package was made for Ruby Sass which not supported anymore, so you'll have to copy the source into your own build-system and remove the `--no-cache` option from the [sublime-build file](https://github.com/jaumefontal/SASS-Build-SublimeText2/blob/master/SASS.sublime-build)**.

## Step-By-Step Solution
I know this is a bit late, but I've just spent four soul-crushing hours figuring out the solution to this really easy problem, and I don't want anyone else to go through the same thing.

### The problem

If you're facing this problem, you've installed [Dart Sass](https://sass-lang.com/dart-sass), which is a **good thing**. Ruby Sass isn't supported anymore. Unfortunately, this package doesn't directly work with Dart Sass.

Basically, this package contains [sublime-build](https://www.sublimetext.com/docs/build_systems.html) files, which contain executable commands to convert sass to css. Ruby Sass had a command line option called [--no-cache](https://sass-lang.com/documentation/cli/ruby-sass#no-cache) which is hard coded into this package's compilation commands. Long story short, Dash Sass doesn't support this flag, so we have to remove it.

### How to solve it

Basically, you'll have to make your own build file without the  --nocache flag. **This is easy.**

1. **In Sublime, go to Tools -> Build System -> New Build System.** This should create a new file called **untitled.sublime-build**, with some default code. Delete it.
2. Go to this repository's [SASS.sublime-build](https://github.com/jaumefontal/SASS-Build-SublimeText2/blob/master/SASS.sublime-build) file and copy all of its contents.
3. Paste it into your file.
4. Remove the  **`, "--no-cache"`**  bit. It's at the end of the line starting with **`"cmd"`**, it should be easy to spot.
5. Save your file as **SASS.sublime-build** in the default location suggested.

That's it. You're all good to go. Use Ctrl + B to build.

You should be able to follow the exact same process for the **SASS - Compressed.sublime-build** file if you want compressed css output, but if you don't know what compressed css is, you probably don't need it.

**PS:** Consider installing [this](https://packagecontrol.io/packages/SublimeOnSaveBuild) package, which builds your css file automatically every time you save.
